### PR TITLE
Authority vaidation integration issue #12 integrate

### PR DESCRIPTION
**Pull request by shivbijlani**

### CODE REVIEW

**Comment by shivbijlani** [comment]
image

@kpanwar , @rohitnarula7176 : please take a look

File: devApps/VanillaJSTestApp/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "msal-js-devapp",
+  "version": "1.1.0",
```
Comment (kpanwar) [inline devApps/VanillaJSTestApp/package.json]: >1.1.0 [](start = 14, length = 5)

shouldn't this be like 1.0.0 or 0.1.0?

Comment (shivbijlani) [inline devApps/VanillaJSTestApp/package.json]: I'll make it 0.1.0

---
In reply to: [114031943](https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/27#discussion_r114031943) [](ancestors = 114031943)

**Comment by kpanwar** [comment]
> /// <reference path="Authority.ts" /> // this is needed to work around error TS2690

We need to add MIT license in all source files. You can track that as a separate issue and PR

---
Refers to: lib/AadAuthority.ts:1 in 59cec7e. [](commit_id = 59cec7e119cf77be3139d5e41a591afdae7994fb, deletion_comment = True)

**Comment by shivbijlani** [comment]
> /// <reference path="Authority.ts" /> // this is needed to work around error TS2690

[https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/31](https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/31)

---
In reply to: [298122651](https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/27#issuecomment-298122651) [](ancestors = 298122651)

---
Refers to: lib/AadAuthority.ts:1 in 59cec7e. [](commit_id = 59cec7e119cf77be3139d5e41a591afdae7994fb, deletion_comment = True)

**Comment by kpanwar** [comment]
:shipit:

**Pull request merged**